Adds deployment cookbook with a set of examples/instructions (#394)
DAGWorks-Inc/burr: docs/examples/deployment/index.rst

✈ Deployment
=============

Burr is specifically meant to make getting your product in production easier and faster.
This section covers examples of getting Burr to production, as well as a brief overview of approaches/requirements.

> To deploy Burr in production, you need to do three things:

Reviewer: ```suggestion
To deploy a Burr application in production, you need to do three things:
```

> 1. Place your application in a web server

Reviewer: ```suggestion
1. Place your Burr application in some place you can trigger it. E.g. a web-service, a script, a library, etc.
```

> 2. Provisioning infrastructure

Reviewer: ```suggestion
2. Provision infrastructure to run (1)
```
